libsrc/telestrat/oserror.s

```diff
@@ -0,0 +1,17 @@
+;
+; Stefan Haubenthal, 2011-04-18
+;
+; int __fastcall__ _osmaperrno (unsigned char oserror);
+; /* Map a system specific error into a system independent code */
+;
+
+        .include        "errno.inc"
+        .export         __osmaperrno
+
+.proc   __osmaperrno
+
+        lda     #<EUNKNOWN
+        ldx     #>EUNKNOWN
```

Comment:
I did initial functions, because it's in my plans to do it, but it's not finished, i removed theses 2 files from my repo. I'll finished it.